sphericart-torch/python/tests/test_nn.py

```diff
@@ -31,9 +31,11 @@ def forward(self, positions):
                 retain_graph=True,
                 create_graph=True,
             )[0]
+            positions.requires_grad = False  # If not, further gradients with respect to xyz will be requested in backward()
```

Comment:
this is weird. Changing require_grad at this point should not change the graph

Reply:
Thanks for confirming. I've removed it